.evergreen/config.in.yml

```diff
@@ -956,29 +878,14 @@ functions:
           export AWS_REGION='${AWS_REGION}'
           export AWS_CMK_ID='${AWS_CMK_ID}'
           EOT
-    - command: shell.exec
+    - command: subprocess.exec
       type: test
       params:
-        working_dir: "src"
-        timeout_secs: 60
-        shell: bash
-        script: |
-          ${PREPARE_SHELL}
-
-          # Disable xtrace (just in case it was accidentally set).
-          set +x
-          source ./prepare_client_encryption.sh
-          rm -f ./prepare_client_encryption.sh
-
-          export VERSION=${VERSION}
-          export DRIVERS_TOOLS=${DRIVERS_TOOLS}
-
-          source ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
-          export MONGODB_URI="${MONGODB_URI}"
-
-          echo "CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
-
-          bash ${PROJECT_DIRECTORY}/.evergreen/run-custom-csfle-tests.sh
+        working_dir: src
```

Comment:
Half of this block was configuring FLE variables, the other half re-exported expansions. It can all be removed.